examples/SampleApp/src/screens/ThreadScreen.tsx

```diff
@@ -51,6 +78,11 @@ export const ThreadScreen: React.FC<ThreadScreenProps> = ({
     return () => setSelectedImages([]);
   }, []);
 
+  useEffect(() => {
+    setSelectedImages([]);
+    return () => setSelectedImages([]);
+  }, []);
```

Comment:
Why do we have this? I can see exact same useEffect on live above

Reply:
Sorry, I guess this was a copy/paste error